loc/LCI/lib/OpenFolderSchema.json.lci

```diff
@@ -0,0 +1,787 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<LCX SchemaVersion="6.0" Name="Z:\src\MIEngine\bin\Desktop.Debug\OpenFolderSchema.json" PsrId="306" FileType="1" SrcCul="en-US" Desc="Commenting file created by LCXAdmin. For more information, please visit http://localizability/longhorn/LcxAdmin.asp" xmlns="http://schemas.microsoft.com/locstudio/2006/6/lcx">
```

Comment:
@danyeh Does that filename matter? This was the path on my local machine generated by the LCXAdmin tool